support/modules-artifact-gen/single-file-artifact-gen

```diff
@@ -178,6 +178,30 @@ else
   stat -c %a "${file}" > $permissions_file
 fi
 
+# Check the the passthrough_args and potentially modify them
+# to avoid conflicts or to let them override the already args
+# provided to mender-artifact
+passthrough_args_modified=" "
+parse=$(xargs -n 2 printf "%s %s\n" <<< $passthrough_args)
```

Comment:
An `sh` compatible alternative is
```suggestion
parse=$(echo $passthrough_args | xargs -n 2 printf "%s %s\n")
```

And similar for the next line.

Or, both combined:
```suggestion
echo $passthrough_args | xargs -n 2 printf "%s %s\n" | while read -r flag arg; do
```